src/Moryx.CommandCenter.Web/src/modules/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -45,6 +45,13 @@ export default class ConfigEditor extends React.Component<ConfigEditorPropModel,
         };
     }
 
+    public componentDidUpdate(prevProps: Readonly<ConfigEditorPropModel>, prevState: Readonly<ConfigEditorStateModel>, snapshot?: any): void {
+        if (this.props.ParentEntry && (prevProps.ParentEntry?.value.current !== this.props.ParentEntry.value.current)) {
+            this.setState({SelectedEntryType: this.props.ParentEntry.value.current});
+
```

Comment:
empty line

Reply:
I just saw that now and will fix it, when I get back to it. But would leave such minor things otherwise until we have updated linter and setup pipeline to report those issues.